### PR TITLE
testing/rocksdb: upgrade to 5.15.10

### PR DESCRIPTION
**Pull request by filoozom**
https://github.com/facebook/rocksdb/releases

cc @jbowes

### CODE REVIEW

File: testing/rocksdb/APKBUILD [hunk 1]
```diff
@@ -1,32 +1,22 @@
 # Contributor: James Bowes <jbowes@repl.ca>
 # Maintainer:
 pkgname=rocksdb
-pkgver=4.2
-pkgrel=2
+pkgver=5.15.10
+pkgrel=3
 pkgdesc="A persistent key-value store for fast storage environments"
 url="http://rocksdb.org"
```
Comment (andypost) [inline testing/rocksdb/APKBUILD]: please update URL to https while you on it

File: testing/rocksdb/APKBUILD [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
+arch="all"
+license="GPL-2.0"
 depends=""
-depends_dev="bzip2-dev snappy-dev zlib-dev"
-makedepends="$depends_dev linux-headers"
+depends_dev="bzip2-dev snappy-dev zlib-dev lz4-dev zstd-dev"
+makedepends="$depends_dev linux-headers bash perl"
 install=""
 subpackages="$pkgname-dev $pkgname-doc"
+options="!check"
```
Comment (andypost) [inline testing/rocksdb/APKBUILD]: it needs comment why tests are disabled

**Comment by filoozom** [comment]
@andypost There's a test suite for source code, but not to check the actual shared library as far as I know. Should I just leave the `!check` in that case?

**Comment by andypost** [comment]
@filoozom yep, makes sense - I used to try locally and no way to test shared lib, moreover if you run `make check` it consumes more the 20Gb of disk

**Comment by andypost** [comment]
it fails on builders
- http://build.alpinelinux.org/buildlogs/build-edge-armhf/testing/rocksdb/rocksdb-5.15.10-r3.log
- http://build.alpinelinux.org/buildlogs/build-edge-aarch64/testing/rocksdb/rocksdb-5.15.10-r3.log
- http://build.alpinelinux.org/buildlogs/build-edge-ppc64le/testing/rocksdb/rocksdb-5.15.10-r3.log

So probably better to disable this arches

**Comment by andypost** [comment]
@filoozom yes, there's easy way https://github.com/alpinelinux/alpine-chroot-install
 wiki
- https://wiki.alpinelinux.org/wiki/Installing_Alpine_Linux_in_a_chroot
- https://wiki.alpinelinux.org/wiki/Install_Alpine_on_LXC

**Comment by andypost** [comment]
Meanwhile I did disable fbc6a681 to not block builders

**Comment by filoozom** [comment]
@andypost I was actually able to build this in a chrooted `aarch64` environment (itself in a Ubuntu Docker container) without any modifications:

```bash
$ uname -a
Linux web02 4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018 aarch64 Linux

$ apk --print-arch
aarch64
```

Is there something special about those builders?

**Comment by andypost** [comment]
@filoozom not sure but there's fix for ppc64le https://github.com/alpinelinux/aports/pull/5519

**Pull request closed without merging**